x-pack/plugins/apm/public/components/app/alerts_overview/index.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useMemo } from 'react';
+import { EuiPanel, EuiFlexItem, EuiFlexGroup } from '@elastic/eui';
+import { ALERT_STATUS } from '@kbn/rule-data-utils';
+import { AlertConsumers } from '@kbn/rule-data-utils';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { ApmPluginStartDeps } from '../../../plugin';
+import { useApmParams } from '../../../hooks/use_apm_params';
+import { SERVICE_NAME } from '../../../../common/elasticsearch_fieldnames';
+import { environmentQuery } from '../../../../common/utils/environment_query';
+import {
+  AlertsTableStatusFilter,
+  ALL_ALERTS_FILTER,
+  AlertStatusFilterButton,
+} from '../../alerting/service_overview_alerts/alerts_table_status_filter';
+
+export function AlertsOverview() {
+  const {
+    path: { serviceName },
+    query: { environment },
+  } = useApmParams('/services/{serviceName}/alerts');
+  const { services } = useKibana<ApmPluginStartDeps>();
+  const [alertStatusFilter, setAlertStatusFilter] = useState(ALL_ALERTS_FILTER);
+
+  const {
+    triggersActionsUi: {
+      getAlertsStateTable,
+      alertsTableConfigurationRegistry,
+    },
+  } = services;
+
+  const alertQuery = useMemo(
+    () => ({
+      bool: {
+        filter: [
+          {
+            bool: {
```

Comment:
Would these not work if they are just `term` filters?

Reply:
filtering the `serviceName`, `term` will work but it won't for the `status` because I need to match all filter statuses for `ALL_ALERTS_FILTER` . Updated [here](https://github.com/elastic/kibana/pull/134350/commits/d4e7ffb66eb273d2ec2016d10ec71a9047bb0c13)

Reply:
let me try it.

Reply:
I know only this helper 
`import { termQuery } from '@kbn/observability-plugin/server';`
but since i'm in  `public` folder i get eslint error

`error  Unexpected path "@kbn/observability-plugin/server" imported in restricted zone. Server modules cannot be imported into client modules or shared modules  @kbn/eslint/no-restricted-paths`

Reply:
> I know only this helper `import { termQuery } from '@kbn/observability-plugin/server';` but since i'm in `public` folder i get eslint error
> 
> `error Unexpected path "@kbn/observability-plugin/server" imported in restricted zone. Server modules cannot be imported into client modules or shared modules @kbn/eslint/no-restricted-paths`

You're right, thanks for updating to terms!